Sources/Splash/Output/AttributedStringOutputFormat.swift

```diff
@@ -94,21 +112,52 @@ private extension NSFont {
             .cfurlposixPathStyle,
             false
         )
-
+        
         guard let font = url.flatMap(CGDataProvider.init).flatMap(CGFont.init) else {
             return nil
         }
-
+        
         return CTFontCreateWithGraphicsFont(font, size, nil, nil)
     }
-
+    
     static func defaultFont(ofSize size: CGFloat) -> NSFont {
         guard let courier = loaded(from: "/Library/Fonts/Courier New.ttf", size: size) else {
             return .systemFont(ofSize: size)
         }
-
+        
         return courier
     }
 }
+#endif
+
+#if os(iOS)
+private extension NSMutableAttributedString {
+    func append(_ string: String, font: UIFont, color: Color) {
+        let color = UIColor(
+            red: CGFloat(color.red),
+            green: CGFloat(color.green),
+            blue: CGFloat(color.blue),
+            alpha: CGFloat(color.alpha)
+        )
+        
+        let attributedString = NSAttributedString(string: string, attributes: [
+            .foregroundColor: color,
+            .font: font
+            ])
+        
+        append(attributedString)
+    }
+}
 
+private extension UIFont {
+    
+    static func defaultFont(ofSize size: CGFloat) -> UIFont {
+        guard let menlo = UIFont(name: "Menlo-Regular", size: size) else {
```

Comment:
Very nice 👍

Reply:
![themes](https://user-images.githubusercontent.com/25137824/44622319-d2e0ca80-a8be-11e8-82f5-927102fa6aa1.png)

I also added a few default themes. Let me know what you think ✌️